pxp/request_response.md

```diff
@@ -117,16 +117,46 @@ messages.
 
 ##### Blocking Response
 
-The results of the requested action should be included in the *results* object;
-its schema is action-specific. *transaction_id* is the other required entry.
+The output of the requested action (results on stdout, errors on stderr, and the
+exit code) should be included in the *output* object.
+
+Note that there's no requirement for the action's *stdout* type, as its format
+is action-specific. As for the agent operation, the *stdout* JSON schema should
+be enforced by the agent in order to guarantee the specific format expected by
```

Comment:
I wonder: Can we say that the format of the stdout is specified by the action metadata for response messages (both blocking & non blocking) and it is a ```string``` for error messages?